isfolkid.sql

```diff
@@ -0,0 +1,181 @@
+-- Hversu margar aðalpersónur eru í bókabálkinum?
+-- 75
+WITH RECURSIVE split(characters, str) AS (
+    SELECT
+        '',
+        characters||','
+    FROM
+        books
+    UNION ALL SELECT
+        TRIM(substr(str, 0, instr(str, ','))),
+        substr(str, instr(str, ',')+1)
+    FROM split
+    WHERE str != ''
+)
+SELECT
+    COUNT(DISTINCT characters)
+FROM split
+WHERE characters != '';
+
+-- Hversu margar persónur eru í hverri bók?
+/*
+Álagafjötrar|2
+Nornaveiðar|3
+Hyldýpið|1
+Vonin|1
+Dauðasyndin|2
+Illur arfur|3
+Draugahöllin|1
+Dóttir böðulsins|2
+Skuggi fortíðar|2
+Vetrarhríð|2
+Blóðhefnd|3
+Ástarfuni|2
+Fótspor Satans|3
+Síðasti riddarinn|2
+Austanvindar|2
+Gálgablómið|3
+Garður dauðans|2
+Gríman fellur|2
+Tennur drekans|1
+Hrafnsvængir|3
+Um óttubil|3
+Jómfrúin og vætturin|3
+Vorfórn|4
+Í iðrum jarðar|2
+Guðsbarn eða galdranorn|3
+Álagahúsið|1
+Hneykslið|2
+Ís og eldur|3
+Ástir Lúcífers|3
+Ókindin|6
+Ferjumaðurinn|2
+Hungur|2
+Martröð|2
+Konan á ströndinni|1
+Myrkraverk|1
+Galdratungl|4
+Vágestur|2
+Í skugga stríðsins|2
+Raddirnar|2
+Fangi tímans|3
+Djöflafjallið|1
+Úr launsátri|1
+Í blíðu og stríðu|1
+Skapadægur|1
+Böðullinn|3
+Svarta vatnið|1
+Er einhver þarna úti?|1
+*/
+
+SELECT
+    is_title,
+    LENGTH(characters) - LENGTH(REPLACE(characters, ',', '')) + 1
+FROM books;
+
+-- Hversu oft kemur Þengill fyrir í bókunum?
+-- 2
+WITH RECURSIVE split(characters, str) AS (
+    SELECT
+        '',
+        characters||','
+    FROM
+        books
+    UNION ALL SELECT
+        TRIM(substr(str, 0, instr(str, ','))),
+        substr(str, instr(str, ',')+1)
+    FROM split
+    WHERE str != ''
+)
+SELECT
+    COUNT(*)
+FROM split
+WHERE characters = 'Þengill';
```

Comment:
Væri hægt að gera einfaldara með 
```suggestion 
SELECT COUNT(*) FROM books WHERE characters LIKE '%Þengill%'
```